src/stores/compliance-ccpa-guide.md

```diff
@@ -0,0 +1,145 @@
+   
+Take inventory of your digital properties, including all integrations and those who have access to your customer data.
+
+  1.  Determine what public and private personal information is collected through your website(s) and mobile application(s).  For example, a standard Magento database stores the following types of public and private personal information: 
+
+      **Public**: Wish Lists, Product Reviews
+
+      **Private**: Customer Information, Order Information, Reward Points, Gift Registry, Address Book, Store Credit, Payment Methods, Billing Agreements, Newsletter Subscriptions, Invitations.
+
+      If your Magento installation has been customized, additional personal information might be collected. Personal information might also reside in [cookies]({% link stores/cookie-reference.md -%}), tags, and other technologies that collect information.
+
+  1.  Identify the parties with whom you share data. Your list will include service providers and third-parties such as advertising networks, internet service providers, data analytics providers, government entities, operating systems and platforms, social networks, and consumer data resellers who do not directly collect personal information from your customers.
+
+      **Service Providers**:  Those who have access to your customer data for a business purpose, and provide services on your behalf. For example, Adobe is service provider, as are some developers of customizations and extensions and services. 
+
+      Check the default settings of Google Universal Analytics, Google Tag Manager &mdash; and any other data services you use &mdash; and make any changes necessary to comply with the regulation. To learn more, see [Google Analytics Settings for GDPR]({% link stores/compliance-gdpr-google.md -%}).  
+
+      **Other Third Parties**: Those with whom you share or sell customer data. For example, you might share customer data with an advertising network in exchange for advertising.
+
+### Step 3: Map the customer journey and data collection process in your store(s).
+
+**Stakeholders:** User Experience, Information Technology, Administrative Support
+
+1.  Identify each point in the [customer journey]({% link getting-started/quick-tour.md -%}) where personal information is collected, and the type of information that is collected at each step.
+
+    Visitors to your site must be notified in advance, or at the point of data collection. For example, a Magento store without custom integrations collects personal information when a customer account is created, and during checkout. If your store has custom integrations, there might be additional data items and attributes to identify. 
+
+1.  Although the following topics were written to help Magento merchants in the EU comply with the GDPR, they can be adapted for CCPA compliance. For data flow diagrams and technical information about database entities, see the topic For your version of Magento.
```

Comment:
This will need to change when you update devdocs. In other words, this will ultimately point to the "statute-agnostic" data flow diagrams once you add a layer of abstraction between GDPR and the data flows.

Reply:
This has been updated to reference the new generic docs.